packages/flutter_analysis_options/lib/analysis_options.yaml

```diff
@@ -0,0 +1,172 @@
+# Specify analysis options.
+#
+# Until there are meta linter rules, each desired lint must be explicitly enabled.
+# See: https://github.com/dart-lang/linter/issues/288
+#
+# For a list of lints, see: http://dart-lang.github.io/linter/lints/
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+#
+# There are other similar analysis options files in the flutter repos,
+# which should be kept in sync with this file:
+#
+#   - analysis_options.yaml (this file)
```

Comment:
This comment probably needs to be updated?

Reply:
I've moved the analysis_options_user stuff to here as well and updated these

Reply:
Updated